### PR TITLE
Fix more LSP tests.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tools/lsp/server/file_server.toit [hunk 1]
```diff
@@ -88,39 +87,39 @@ class FileServerProtocol:
...
-          file := get_file path
+          file := get_file compiler_path
           encoded_size := file.content == null ? -1 : file.content.size
           encoded_content := file.content == null ? "" : file.content
           writer.write "$file.exists\n$file.is_regular\n$file.is_directory\n$encoded_size\n"
           writer.write encoded_content
 
-  get_file path /string -> File:
-    return file_cache_.get path --init=: create_file_entry_ path
+  get_file compiler_path /string -> File:
```
Comment (kasperl) [inline tools/lsp/server/file_server.toit:104]:
```suggestion
  get_file compiler_path/string -> File:
```

or maybe:

```suggestion
  get_file compiler_path / string -> File:
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tests/lsp/export_summary_compiler_test.toit [hunk 2]
```diff
@@ -27,7 +27,8 @@ check_foo_export summary client foo_path:
 
 test client/LspClient:
   LEVELS ::= 3
-  DIR ::= "/non_existing_dir_toit_test"
+  DRIVE ::= platform == PLATFORM_WINDOWS ? "C:" : ""
```
[the remark this answers is not in the document]
Comment (floitsch) [reply tests/lsp/export_summary_compiler_test.toit]: done.

File: tools/lsp/server/file_server.toit (hunk 1 above)
Comment (floitsch) [reply tools/lsp/server/file_server.toit]: done.

**Pull request merged**